app/views/history/index.html.erb

```diff
@@ -20,7 +27,16 @@
         <tbody>
         <% @results.each do |result| %>
           <tr>
-            <td class="player"><%= player_label_for_result result, card_history_additions(result.player_card_histories) %></td>
+            <td class="choose">
+              <% if result.arena? %>
+                <%= check_box_tag "result_ids[]", result.id, false, class: "bulk-edit-control bulk-edit-picker", title: "Arena results cannot be edited currently", disabled: true%>
```

Comment:
May not even need the result id attributes for arena results if we're not allowing deletion of them yet? Even if user modifies the DOM and enables the field they still won't be able to delete the result because of the checks I put in the model.